ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -497,7 +498,28 @@ bool Ogre2SelectionBuffer::ExecuteQuery(const int _x, const int _y,
     auto collection = this->dataPtr->sceneMgr->findMovableObjects(
         Ogre::ItemFactory::FACTORY_TYPE_NAME, entName);
     if (collection.empty())
+    {
+      // try heightmaps
+      auto heightmaps = this->dataPtr->scene->Heightmaps();
+      for (auto h : heightmaps)
+      {
+        auto heightmap = h.lock();
+        if (heightmap)
+        {
+          if (entName == heightmap->Name())
+          {
+            // \todo(anyone) change return type to MovableObject instead of item
```

Comment:
is this something that we should do for `gz-rendering8` ?

Reply:
yes added note about gz-rendering8 092dd0cdfb7197eeb85a2e894b6f58c7f68f2888